tests/geometryXVx/CMakeLists.txt

```diff
@@ -41,6 +40,29 @@ gtest_discover_tests(unit_tests_${GEOMETRY_VARIANT}
     DISCOVERY_MODE PRE_TEST
 )
 
+add_executable(unit_tests_maxwellian_${GEOMETRY_VARIANT}
+    maxwellian.cpp
+    ../main.cpp
+)
+
+target_link_libraries(unit_tests_maxwellian_${GEOMETRY_VARIANT}
+    PUBLIC
+        GTest::gtest
+        GTest::gmock
+        gslx::geometry_${GEOMETRY_VARIANT}
+        gslx::initialisation_${GEOMETRY_VARIANT}
+        gslx::quadrature
+        gslx::speciesinfo
+        gslx::time_integration_${GEOMETRY_VARIANT}
+        gslx::utils
+)
+
+gtest_discover_tests(unit_tests_maxwellian_${GEOMETRY_VARIANT}
+    TEST_SUFFIX "_${GEOMETRY_VARIANT}"
+    PROPERTIES TIMEOUT 10
+    DISCOVERY_MODE PRE_TEST
+)
+
```

Comment:
Why is this change necessary?

Reply:
> - isolate the maxwellian tests to avoid failures, not clear yet why this makes it work. The issue is being tracked by https://github.com/CExA-project/ddc/issues/790